src/www/firewall_scrub_edit.php

```diff
@@ -422,7 +422,7 @@ function formNetworks()
                   <tr>
                     <td><a id="help_for_srcport" href="#" class="showhelp"><i class="fa fa-info-circle"></i></a> <?=gettext("Source port"); ?></td>
                     <td>
-                      <div style="display:hidden;">
+                      <div style="visibility:hidden;">
```

Comment:
better use display: none or class="hidden" which is more common in our code

Reply:
Fixed.